index.md

```diff
@@ -37,6 +37,30 @@ consumers of your library:
 A scope may be provided to a commit's type, to provide additional contextual information and
 is contained within parenthesis, e.g., `feat(parser): add ability to parse arrays`.
 
+## Examples
+
+### Commit message with body and breaking change
+```
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+
+### Commit message with no body
+```
+docs: correct spelling of CHANGLOG
+```
+
+### Commit message with scope
+```
+feat(lang): added polish language
```

Comment:
WDYT about using imperative sentence in the examples?
This is a shared best practice (also in the angular repo)

`feat(lang): add polish language`

Reply:
If it's not enforced by this convention, I would ask you to open a new issue to discuss this.

Reply:
I second this. As I mentioned on my other comment, even if it's not in the spec we shouldn't have "bad" commit messages in the examples. Especially if people would end up relying on tools to enforce them that'd scream at this.

Reply:
We should have "bad" commit messages if the spec doesn't require "good" ones. By trying to include best practices from other tools we'll end up constantly chasing any changes in those tools / conventions. Best to describe those other conventions in one place only and keep it DRY. In any case, the discussion should really go in a new issue.

Reply:
Created stub issue: https://github.com/conventional-changelog/conventionalcommits.org/issues/42